Products/CMFCore/exportimport/content.py

```diff
@@ -23,6 +23,7 @@
 from six.moves.configparser import ConfigParser
 
 from DateTime import DateTime
+from Products.CMFCore.interfaces import ITypesTool
```

Comment:
please revert this

Reply:
I will revert this.  For the record, I let this change sneak into my commit because in the Plone codebase, we decided to adopt the guideline to only use absolute imports for Python 3 compatibility, with no explicitly relative imports.
Does Zope have written coding conventions in this regard?

Reply:
There is no written policy but this is about consistency. If you look around the package you will consistently see relative imports used.